### DISCUSSION TITLE
Nuxt 3

### DISCUSSION THREAD

**Discussion by MyWay** [umputun/remark42#1374] [Ideas]
Hi, would it be possible to add a guide to integrate with Nuxt 3, like the one for Gatsby? I'm not getting any error, but che component doesn't show.

**Comment by akellbl4** [comment]
@MyWay hey, good idea.
Now I'm working on new SPA API. If you want to have manual on current API we could do it together. You can publish your example on GitHub and I'll try to fix the problems. WDYT?

**Reply by MyWay** [reply]
Hi @akellbl4! Seems it's working now, but feel free to have a look and do things better: https://github.com/MyWay/nuxt3-remark42

**Reply by akellbl4** [reply]
I've checked your code.
I'd recommend to inline the remark42 script. The inline script is small and it loads scripts for modules with `defer` which shouldn't affect loading performance.
